### DISCUSSION TITLE
Different aiming points for projectiles and turrets in multiplayer mode

### DISCUSSION THREAD

**Discussion by jcx1991911** [FakeFishGames/Barotrauma#14096] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I built a multi-turret focus system with the new distance_out pin and it works fine in single player like this:

https://github.com/FakeFishGames/Barotrauma/assets/116519004/82eb9f04-4442-469b-8a16-135e19c644fe

but in multiplayer, it look like this:

https://github.com/FakeFishGames/Barotrauma/assets/116519004/d55c6d62-5a64-419f-bb54-9f719e573663

You can clearly see that the left turret's shots are clearly off its angle
Here's my test sub:
[Sea Scorpion M3.zip](https://github.com/user-attachments/files/15741773/Sea.Scorpion.M3.zip)

### Reproduction steps

1 host a multiplayer game
2 use the sub I upload
3 mount the coligun and shoot

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Addressed in https://github.com/FakeFishGames/Barotrauma-development/commit/e7091bd10993cdadcd371813db512cd7499fd58d by reverting the addition of the distance_out pin.

The pin was added from a community pull request, and we did not realize the approach doesn't work at all in multiplayer because the exact position of the player's cursor isn't synced, just the direction they're aiming towards.

**Reply by girlyguppy** [reply]
Can we have it back
